tasks/clean/remove_invalid_filtered_master_domains.yml

```diff
@@ -1,11 +1,11 @@
 - block:
-    - name: Fetch storage domains for detach
+    - name: Fetch active/maintenance/detached storage domain for remove
       ovirt_storage_domains_facts:
-          pattern:  name={{ storage['dr_' + dr_source_map + '_name'] }} and {{ dr_query_domain_search }}
+          pattern:  name={{ storage['dr_' + dr_source_map + '_name'] }} and {{ dr_inactive_domain_search }}
           auth: "{{ ovirt_auth }}"
 
     - name: Remove storage domain
-      include_tasks: tasks/clean/remove_domain_process.yml sd={{ item }}
+    - include_tasks: tasks/clean/remove_domain_process.yml sd={{ sd }}
```

Comment:
I think `-` shouldn't be here before `include_tasks` there is ` - name` already

Reply:
yet, good catch

Reply:
done